### ISSUE TITLE
Accessible buffer error on startup

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#181541]
Seeing this when running ./scripts/code on Windows:

```
[29052:0504/092953.790:INFO:CONSOLE(244)] "%c  ERR color: #f33 Cannot read properties of undefined (reading 'minimapWidth'): TypeError: Cannot read properties of undefined (reading 'minimapWidth')
    at EditorScrollbar._setLayout (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/viewParts/editorScrollbar/editorScrollbar.js:80:93)
    at new EditorScrollbar (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/viewParts/editorScrollbar/editorScrollbar.js:42:18)
    at new View (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/view.js:34:31)
    at CodeEditorWidget._createView (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:1325:26)
    at CodeEditorWidget._attachModel (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:1238:46)
    at CodeEditorWidget.setModel (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:300:18)
    at new SuggestEnabledInput (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/contrib/codeEditor/browser/suggestEnabledInput/suggestEnabledInput.js:66:30)
    at InstantiationService._createInstance (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/platform/instantiation/common/instantiationService.js:91:28)
    at InstantiationService.createInstance (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/platform/instantiation/common/instantiationService.js:62:31)
    at ExtensionsViewPaneContainer.create (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/contrib/extensions/browser/extensionsViewlet.js:415:71)
    at PaneContainer.create (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/panecomposite.js:30:36)
    at SidebarPart.showComposite (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/compositePart.js:132:27)
    at SidebarPart.doOpenComposite (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/compositePart.js:73:18)
    at SidebarPart.openComposite (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/compositePart.js:46:25)
    at SidebarPart.doOpenViewlet (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/sidebar/sidebarPart.js:177:25)
    at SidebarPart.openPaneComposite (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/sidebar/sidebarPart.js:141:29)
    at PaneCompositeParts.openPaneComposite (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/parts/paneCompositePart.js:38:66)
    at vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/browser/layout.js:604:65
    at async Promise.all (index 2)
    at async Object.settled (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/base/common/async.js:1129:28)", source: vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/platform/log/common/log.js (244)
[29052:0504/092953.839:INFO:CONSOLE(248)] "%c[Extension Host] %cdebugger listening on port 5870 color: blue color:", source: vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/services/extensions/electron-sandbox/localProcessExtensionHost.js (248)
[29052:0504/092954.019:INFO:CONSOLE(179)] "%c[Extension Host] %cdebugger inspector at devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:5870/4a3675ef-24bf-4662-bf9b-9700a2d6c811 color: blue color:", source: vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/services/extensions/electron-sandbox/localProcessExtensionHost.js (179)
[29052:0504/092954.520:INFO:CONSOLE(244)] "%c  ERR color: #f33 Cannot read properties of undefined (reading 'minimapWidth'): TypeError: Cannot read properties of undefined (reading 'minimapWidth')
    at EditorScrollbar._setLayout (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/viewParts/editorScrollbar/editorScrollbar.js:80:93)
    at new EditorScrollbar (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/viewParts/editorScrollbar/editorScrollbar.js:42:18)
    at new View (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/view.js:34:31)
    at CodeEditorWidget._createView (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:1325:26)
    at CodeEditorWidget._attachModel (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:1238:46)
    at CodeEditorWidget.setModel (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/editor/browser/widget/codeEditorWidget.js:300:18)
    at AccessibleBufferWidget._updateModel (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.js:207:31)
    at async AccessibleBufferWidget.updateEditor (vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.js:164:27)", source: vscode-file://vscode-app/c:/Github/microsoft/vscode/out/vs/platform/log/common/log.js (244)
```

**Comment by Tyriar** [comment]
When I try opening a new file 😕 

![image](https://user-images.githubusercontent.com/2193314/236269996-006f25d5-5c4e-4a48-bc99-7029a9b32e80.png)

Doing a clean build to verify it's not a problem on my end

**Comment by Tyriar** [comment]
This was a stale build problem, went away when I cleaned and restarted the build
